### PR TITLE
RO migration to ODK 1.4

### LINKED ISSUE #563: Move RO build to ODK
As suggested by @matentzn in Slack

### PR DESCRIPTION
**Pull request by gouttegd**
Fixes #563

- chemicals where merged into EDIT

### CODE REVIEW

File: src/metadata/ro.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: ontology_detail
+id: ro
+title: OBO Relations Ontology
+jobs:
+  - id: https://travis-ci.org/oborel/obo-relations
```
Comment (cthoyt) [inline src/metadata/ro.md:6]: Travis-CI seems a bit outdated, no?

Comment (matentzn) [reply src/metadata/ro.md:6]: Please Charlie. Please. Do not look at people PR drafts.. this is like checking someone out while they are getting dressed.

File: core.owl [hunk 2]
```diff
@@ -635,27 +647,153 @@ A continuant cannot have an occurrent as part: use &apos;participates in&apos;.
...
             </owl:Restriction>
         </owl:disjointWith>
-    </rdf:Description>
+        <obo:IAO_0000115 xml:lang="en">An entity that has temporal parts and that happens, unfolds or develops through time.</obo:IAO_0000115>
+        <rdfs:label xml:lang="en">occurrent</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/BFO_0000004 -->
```
Comment (matentzn) [inline core.owl:679]: Why does core have so many BFO classes in it?

File: ro-base.obo [hunk 3]
```diff
@@ -130,6 +122,12 @@ id: GO:0016301
 [Term]
 id: GO:0019107
 
+[Term]
+id: OGMS:0000031
```
Comment (matentzn) [inline ro-base.obo:126]: Why does this show up in ro-base? RO is not really a base it seems.

File: annotations.owl [hunk 4]
```diff
@@ -1,338 +0,0 @@
...
-This interpretation is *not* the same as an at-all-times relation
-
-</obo:IAO_0000115>
-    </owl:NamedIndividual>
-    
-
-
-    <!-- http://purl.obolibrary.org/obo/RO_0001902 -->
-
-    <owl:NamedIndividual rdf:about="&obo;RO_0001902">
```
Comment (matentzn) [inline annotations.owl:269]: This individual seems lost.

File: ro-base.obo (hunk 3 above)
Comment (matentzn) [reply ro-base.obo]: Fixed.

File: annotations.owl (hunk 4 above)
Comment (matentzn) [reply annotations.owl:269]: Added it to EDIT

File: core.owl (hunk 2 above)
Comment (matentzn) [reply core.owl:679]: Because imports are merged. all good

**Review by matentzn — APPROVED** [review]
Excellent work @anitacaron @gouttegd 

I think this is done.

**Pull request merged**